pkg/beholder/client.go

```diff
@@ -239,6 +240,11 @@ func newOtelResource(cfg Config) (resource *sdkresource.Resource, err error) {
 // Open question: what are pros/cons for using use map[]any vs use otellog.KeyValue
 func (e messageEmitter) Emit(ctx context.Context, body []byte, attrKVs ...any) error {
 	message := NewMessage(body, attrKVs...)
+
+	if _, ok := message.Attrs["timestamp"]; !ok {
+		message.Attrs["timestamp"] = time.Now().UTC()
+	}
```

Comment:
It looks like `log.Record`s already have a https://pkg.go.dev/go.opentelemetry.io/otel/log#Record.Timestamp and an https://pkg.go.dev/go.opentelemetry.io/otel/log#Record.ObservedTimestamp. Could we lever those built-ins instead?

Reply:
yep, I think I mentioned that somewhere in other PR previously

Reply:
Done

Reply:
I expected this to be a much more simplifying change. Why do we still need to pass this around as metadata/attribute if we are using the built-in field?